nixos/modules/services/web-servers/nginx/default.nix

```
@@ -529,6 +527,18 @@ in
         '';
       };
 
+      defaultMimeTypes = mkOption {
+        type = types.str;
+        default = "${pkgs.mailcap}/etc/nginx/mime.types";
+        defaultText = literalExpression "$''{pkgs.mailcap}/etc/nginx/mime.types";
+        example = literalExpression "$''{pkgs.nginx}/conf/mime.types";
+        description = lib.mdDoc ''
+          The mime type definitions included with nginx are very incomplete, so
```

The description seems strange, here's my proposal:
```suggestion
          Default MIME types for NGINX, as MIME types definitions from NGINX are very incomplete,
          we use by default the ones bundled in the mailcap package, used by most of the other Linux distributions.
```

---

Fixed.